src/main.rs

```diff
@@ -282,14 +267,10 @@ macro_rules! tryc {
 }
 
 fn get_includes_from_linker_script(linker_script: &str) -> Vec<&str> {
-    let mut includes = vec![];
-    for mut line in linker_script.lines() {
-        line = line.trim();
-        line = eat!(line, "INCLUDE");
-        includes.push(line);
-    }
-
-    includes
+    linker_script
+        .lines()
+        .filter_map(|line| line.trim().strip_prefix("INCLUDE").map(str::trim))
```

Comment:
Why `filter_map` and not just `map`?

Reply:
Because `eat!` calls `continue` if `line` does not start with `"INCLUDE"`, which is equivalent to filtering the list.